config/externalnamenottested.go

```diff
@@ -334,36 +324,20 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	"aws_vpc_endpoint_policy": config.IdentifierFromProvider,
 	// No import
 	"aws_vpc_endpoint_security_group_association": config.IdentifierFromProvider,
-	// IPAMs can be imported using the ipam id
-	"aws_vpc_ipam": config.IdentifierFromProvider,
 	// IPAMs can be imported using the delegate account id
 	"aws_vpc_ipam_organization_admin_account": config.ParameterAsIdentifier("delegated_admin_account_id"),
-	// IPAMs can be imported using the ipam pool id
-	"aws_vpc_ipam_pool": config.IdentifierFromProvider,
 	// IPAMs can be imported using the <cidr>_<ipam-pool-id>
-	"aws_vpc_ipam_pool_cidr": config.IdentifierFromProvider,
```

Comment:
Hi @dverveiko,
I think this was the resource (`aws_vpc_ipam_pool_cidr`) whose configuration was missing in https://github.com/upbound/provider-aws/issues/219. I think we have accidentally overwritten the key.

Reply:
Got it. Thanks!